src/utils/js.ts

```diff
@@ -0,0 +1,11 @@
+export function getArrayDuplicate<T>(array:T[]):T | undefined {
+  const seenValues: any= {}
+
+  for (const entry of array) {
+    if (seenValues[entry]) {
+      return entry;
+    } else {
+      seenValues[entry] = true
+    }
+  }
```

Comment:
A `Set` is probably more appropiate for this

Reply:
Ok